be/src/util/arena.cpp

```diff
@@ -59,6 +60,7 @@ char* Arena::AllocateAligned(size_t bytes) {
 
 char* Arena::AllocateNewBlock(size_t block_bytes) {
     char* result = new char[block_bytes];
+    memset(result, 0, block_bytes);
```

Comment:
Why memset this to 0? This will cost CPU, we should avoid unnecessary memset

Reply:
initialing column_zone_map.max_value will call this method for allocating memory,and then column_zone_map.max_value will be reinterpret_cast to slice(FieldTypeTraits<OLAP_FIELD_TYPE_CHAR>.set_to_min,types.h),if not set to 0,the slice.size value will be unknown(usually very large value), it will cause core dump;
maybe I  cast column_zone_map.max_value to slice and set to 0 instead of  memset to 0 is better

Reply:
Yes, I think it's better to only fix column_zone_map's problem, because this is used in many other code in which memset is unnecessary.

Reply:
👌